### PR TITLE
feat: added preview projection type

### PR DESCRIPTION
**Pull request by cazala**
Added preview projection type

### CODE REVIEW

File: src/dapps/preview/preview-projection.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  generateLazyValidator,
+  ValidateFunction
+} from '../../validation'
+
+export enum PreviewProjection {
+  ORTHOGRAPHIC = 'orthographic',
+  PERSPECTIVE = 'perspective'
+}
+
+/** @alpha */
```
Comment (fzavalia) [inline src/dapps/preview/preview-projection.ts:12]: What is this alpha thingy?

**Review by fzavalia — COMMENTED** [review]
Just a small question

File: src/dapps/preview/preview-projection.ts (hunk 1 above)
Comment (cazala) [reply src/dapps/preview/preview-projection.ts]: lol copypasta, removed

**Comment by fzavalia** [comment]
image

**Review by fzavalia — APPROVED** [review]

**Pull request merged**